### PR TITLE
Enforce python3 for the dockerfile

### PR DESCRIPTION
**Pull request by mvalkon**
This PR converts all uses of python to python3 in the Dockerfile, and
converts each respective python file to be compatible with python3.

There is no backwards compatibility for python versions here, in case
such is needed a more sophisticated solution is needed.

References #80

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -2,7 +2,8 @@ autopep8==1.3.3
 boto3==1.4.7
 coverage==4.4.1
 enum34==1.1.6
-flake8>=2.5.1,<2.6
+flake8==3.5.0
+pycodestyle<2.4.0
```
Comment (mvalkon) [inline requirements.txt]: This is unfortunately needed due to https://gitlab.com/pycqa/flake8/issues/415

File: src/tests/test_bootstrap.py [hunk 2]
```diff
@@ -5,6 +5,7 @@
 import mock
 
 import requests_mock
+import builtins
```
Comment (scherniavsky) [inline src/tests/test_bootstrap.py]: Could you please check the issue reported by Codacy (unused import)?

Comment (mvalkon) [reply src/tests/test_bootstrap.py]: Seems like Codacy can only analyse actual usages. This is used in the [mock](https://github.com/zalando-incubator/docker-locust/pull/87/files#diff-9c056b1bba9a05022f4af92ea0b8bb8eR67).. In python2, the builtins were available without an import but unfortunately that is not the case in py3. Not immediately obvious how to fix this. I'll try to see but is it possible to ignore as a last result?

Comment (mvalkon) [reply src/tests/test_bootstrap.py]: Ah, turns out that for the mock the import is not needed.

**Comment by thilp** [comment]
:+1:

**Comment by scherniavsky** [comment]
@mvalkon Unfortunately I've got an issue with a new version today and had to revert the changes. The issue is:
```
Traceback (most recent call last):
  File "/usr/local/bin/locust", line 7, in <module>
    from locust.main import main
  File "/usr/local/lib/python3.5/dist-packages/locust/main.py", line 14, in <module>
    from . import events, runners, web
  File "/usr/local/lib/python3.5/dist-packages/locust/runners.py", line 13, in <module>
    from gevent.pool import Group
  File "/usr/local/lib/python3.5/dist-packages/gevent/pool.py", line 27, in <module>
    from gevent._imap import IMap
  File "/usr/local/lib/python3.5/dist-packages/gevent/_imap.py", line 227, in <module>
    import_c_accel(globals(), 'gevent.__imap')
  File "/usr/local/lib/python3.5/dist-packages/gevent/_util.py", line 105, in import_c_accel
    mod = importlib.import_module(cname)
  File "/usr/lib/python3.5/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "src/gevent/_queue.pxd", line 18, in init gevent.__imap
ValueError: gevent._queue.Queue has the wrong size, try recompiling. Expected 80, got 72
```

**Comment by mvalkon** [comment]
@scherniavsky That is a little confusing, I don't understand how it is using python 3.5 given that the base ubuntu image I upgraded to ships with python 3.6. I will take a look at this.

**Pull request merged**